app/src/main/kotlin/com/mapzen/erasermap/presenter/MainPresenter.kt

```diff
@@ -2,12 +2,15 @@ package com.mapzen.erasermap.presenter
 
 import com.mapzen.erasermap.view.ViewController
 import com.mapzen.pelias.gson.Result
+import com.mapzen.valhalla.Route
 import com.squareup.otto.Bus
 
 public interface MainPresenter {
     public var currentSearchTerm: String?
     public var viewController: ViewController?
     public var bus: Bus?
+    public var route: Route?
+    public var routingEnabled: Boolean
```

Comment:
Do we need a separate boolean flag or can we just assume routing is enabled when the route is non-null?